src/applications/pensions/config/chapters/04-household-information/helpers.jsx

```diff
@@ -35,6 +36,17 @@ export function showSpouseAddress(formData) {
   );
 }
 
+export function isBetween18And23(childDOB) {
+  return moment(childDOB).isBetween(
```

Comment:
Has there been any talk of deprecating moment js?
Luxon

Reply:
I haven't heard anything about it, but it looks like Luxon would be a better choice after reading up on it.